l10n_be_mis_reports/__openerp__.py

```diff
@@ -39,6 +39,7 @@
         'data/mis_report_styles.xml',
         'data/mis_report_pl.xml',
         'data/mis_report_bs.xml',
+        'data/mis_report_vat.xml',
     ],
     'installable': True,
 }
```

Comment:
Now that we depends on the menu (l10n_be.account_reports_be_statements_menu) we need to add the dependency on l10n_be here.